test/integration/extension.test.ts

```diff
@@ -26,7 +27,7 @@ import { goPlay } from '../../src/goPlayground';
 import { runFillStruct } from '../../src/goFillStruct';
 
 suite('Go Extension Tests', () => {
-	const gopath = process.env['GOPATH'];
+	const gopath = process.env['GOPATH'] || path.join(os.homedir(), 'go');
```

Comment:
Turns out I have function that does all the required fallbacks :)

```suggestion
	const gopath = getCurrentGoPath();
```